dataclients/kubernetes/routegroup.go

```diff
@@ -513,49 +518,120 @@ func transformRouteGroup(ctx *routeGroupContext) ([]*eskip.Route, error) {
 	return explicitGroupRoutes(ctx)
 }
 
+func splitHosts(hosts []string, domains []string) (map[string]string, []string) {
+	i := map[string]string{}
+	e := []string{}
```

Comment:
not sure about `e`, maybe `a` ?

Reply:
it's it from internal and e for external. Will fix it.